### PR TITLE
Add customizable data path

### LINKED ISSUE #1: config path for .yaml file
it would be great to define in the `jekyll contentful` command a variable with the path where the `.yaml` will be generated, or use the `_config.yml` for that

I have the next line in my Jekyll config:
```
source: '_app'
```

so, after running the command I need to copy the resulting file

### PR DESCRIPTION
**Pull request by dlitvakb**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ contentful:
           :preview: false
           :raise_errors: true
           :dynamic_entries: :auto
+        base_path: app_path        # Optional - Defaults to Current directory
```
Comment (neonichu) [inline README.md]: We should use a symbol for consistency with other options.

File: lib/jekyll-contentful-data-import/data_exporter.rb [hunk 2]
```diff
@@ -33,14 +39,19 @@ def destination_file
       end
 
       def setup_directory
-        data_folder = File.join(Dir.pwd, DATA_FOLDER)
-        Dir.mkdir(data_folder) unless Dir.exist?(data_folder)
+        directories_to_create = destination_directory.gsub(Dir.pwd, '')[1..-1].split(File::SEPARATOR)
```
Comment (neonichu) [inline lib/jekyll-contentful-data-import/data_exporter.rb]: Any particular reason why we aren't using `FileUtils.mkdir_p` here?

File: README.md (hunk 1 above)
Comment (dlitvakb) [inline README.md]: Only CDA options are symbols, though I just noticed that my specs use `base_path` as a symbol and it should be a string, fixing that

File: lib/jekyll-contentful-data-import/data_exporter.rb (hunk 2 above)
Comment (dlitvakb) [inline lib/jekyll-contentful-data-import/data_exporter.rb]: Didn't know about that! TIL :+1:

File: README.md (hunk 1 above)
Comment (neonichu) [inline README.md]: :ok_hand:

File: lib/jekyll-contentful-data-import/data_exporter.rb [hunk 3]
```diff
@@ -33,14 +39,15 @@ def destination_file
...
+        FileUtils.mkdir_p destination_directory
+      end
 
-        contentful_folder = File.join(data_folder, CONTENTFUL_FOLDER)
-        Dir.mkdir(contentful_folder) unless Dir.exist?(contentful_folder)
+      private
 
-        spaces_folder = File.join(contentful_folder, SPACES_FOLDER)
-        Dir.mkdir(spaces_folder) unless Dir.exist?(spaces_folder)
+      def create_folder(path, name)
```
Comment (neonichu) [inline lib/jekyll-contentful-data-import/data_exporter.rb]: I believe this method is now unused

Comment (dlitvakb) [inline lib/jekyll-contentful-data-import/data_exporter.rb]: Removed

**Comment by neonichu** [comment]
:shipit:

**Pull request merged**